go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/influxdata/kapacitor v1.5.3
 	github.com/influxdata/usage-client v0.0.0-20160829180054-6d3895376368
 	github.com/jessevdk/go-flags v1.4.0
+	github.com/kevinburke/go-bindata v3.16.0+incompatible // indirect
```

Comment:
run `go mod tidy` so this will be removed, as it's not a project dependency, but a build dependency.